src/stoobly.ts

```diff
@@ -18,6 +19,12 @@ export default class Stoobly {
 
     this.cypress = new Cypress(this.interceptor);
     this.playwright = new Playwright(this.interceptor);
+
+    // Auto-detect and set test name if available
+    const testName = getTestName();
```

Comment:
Shouldn't `testName` be set dynamically when a request is intercepted? Otherwise how does this change depending on the test?